pages/shared/groupWorkInitial.ejs

```diff
@@ -4,7 +4,7 @@
       <div class="col-sm bg-light py-4 px-4 border">
         <form id="create-form" name="create-form" method="POST">
           <h6>Group name</h6>
-          <input type="text" class="form-control" id="groupNameInput" name="groupName" placeholder="e.g. teamOne" aria-describedby="groupNameHelp">
+          <input type="text" class="form-control" id="groupNameInput" name="groupName" maxlength="30" placeholder="e.g. teamOne" aria-describedby="groupNameHelp">
           <small id="groupNameHelp" class="form-text text-muted">Group names can only contain letters and numbers</small>
```

Comment:
Can you update this helper text and the error message that's shown when the group name is invalid? We need to make sure that users actually know that their group name is invalid.

Reply:
My thought was that the HTML maxlength modifier is sufficient UX-wise to indirectly tell user that the length is capped (as when group name reach 30 characters, the user can no longer type in more characters)

But indeed, it can be better to explicitly inform user that the length is capped, hence I made a new commit to update the helper text & error message.